server/function_contexts.h

```diff
@@ -66,9 +66,9 @@ struct ExtractContext: public SqlBindingScope {
 
 struct FunctionExpressionContext: public ReadThroughBindingContext {
 
-    struct RowContext: public SqlRowScope {
-        RowContext(const FunctionContext & input)
-            : input(input)
+    struct RowContext: public ReadThroughBindingContext::RowContext {
```

Comment:
This might be the problem I am seeing with my changes that segfault on some function calls.